src/renderer/ssf-api.ts

```diff
@@ -236,6 +237,28 @@ export class SSFApi {
         }
     }
 
+    /**
+     * Allows JS to register a log retriever that can be used by electron main process.
+     * @param  {Object} retriever  function that can be called to retrieve logs
```

Comment:
No need to put the type in the jsdoc comment, since it is typescript the type can be taken from the method signature

Reply:
done